agent/gpu/nvidia_gpu_manager_unix.go

```diff
@@ -78,6 +79,7 @@ func (n *NvidiaGPUManager) Initialize() error {
 		n.SetGPUIDs(gpuIDs)
 		n.SetDevices()
 	}
+	seelog.Info("Config for GPU support is enabled, but GPU information is not found; continuing without it")
```

Comment:
wondering if this should be handled as a failure, for example when customer is expecting to use GPU, shouldn't the customer be notified of the misconfiguration in a more explicit way?

Reply:
This config is not recommended for customers to add. This is a GPU instance level config, but we also let customers override it to true/false. If for some reason, a customer sets it to true on non GPU instance, there is value is not stopping agent and continuing to register as a normal instance. I can change the log level to error.